### PR TITLE
should you use prisma

### PR DESCRIPTION
**Pull request by nikolasburk**

### CODE REVIEW

File: content/200-concepts/050-overview/250-should-you-use-prisma.mdx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+**Prisma likely is not a good fit for you if ...**
+
+... you need full control over all database workflows
+
+... you do not want to write any code for your backend
+
+... you want a CRUD GraphQL API without writing any code
+
+... you want to use type-safe SQL for querying your database
```
Comment (2color) [inline content/200-concepts/050-overview/250-should-you-use-prisma.mdx]:
```suggestion
... you want to use type-safe plain SQL for querying your database
```

Just so it's not misunderstood.

Comment (nikolasburk) [reply content/200-concepts/050-overview/250-should-you-use-prisma.mdx]: nice catch 🙏

File: content/200-concepts/050-overview/250-should-you-use-prisma.mdx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+### ... you want a CRUD GraphQL API without writing any code
+
+While tools like the [`nexus-plugin-prisma`](https://nexusjs.org/docs/plugins/prisma/overview) and [`typegraphql-prisma`](https://github.com/MichalLytek/typegraphql-prisma#readme) allow you to quickly generate CRUD operations for your Prisma models in a GraphQL API, these approaches still require you to set up your GraphQL server manually and do some work to expose GraphQL queries and mutations for the models defined in your Prisma schema.
+
+If you want to get a GraphQL endpoint for your database out-of-the box, other tools might be better suited for your use case.
+
+**Alternatives**: [Hasura](https://hasura.io/), [Postgraphile](https://www.graphile.org/postgraphile/)
+
+### ... you want to use type-safe SQL for querying your database
```
Comment (nikolasburk) [inline content/200-concepts/050-overview/250-should-you-use-prisma.mdx]:
```suggestion
### ... you want to use raw, type-safe SQL for querying your database
```

File: content/200-concepts/050-overview/250-should-you-use-prisma.mdx [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+**Prisma likely is not a good fit for you if ...**
+
+... you need full control over all database workflows
+
+... you do not want to write any code for your backend
+
+... you want a CRUD GraphQL API without writing any code
+
+... you want to use type-safe raw SQL for querying your database
```
Comment (nikolasburk) [inline content/200-concepts/050-overview/250-should-you-use-prisma.mdx]:
```suggestion
... you want to use raw, type-safe SQL for querying your database
```

File: content/200-concepts/050-overview/250-should-you-use-prisma.mdx [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+Prisma is a new kind of ORM that - like any other tool - comes with its own tradeoffs. This page explains the tradeoffs and mentions which tools can be used as alternatives to Prisma in different scenarios.
+
+</TopBlock>
+
+## TLDR
+
+Here's an overview of the different scenarios where Prisma might or might not be a good fit for your project. To learn about each scenario in more detail, find the respective section further down on this page.
+
+**Prisma likely is a good fit for you if ...**
```
Comment (nikolasburk) [inline content/200-concepts/050-overview/250-should-you-use-prisma.mdx]:
```suggestion
**Prisma likely _is_ a good fit for you if ...**
```

File: content/200-concepts/050-overview/250-should-you-use-prisma.mdx [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+... you are working in a team
+
+... you want a tool that holistically covers your database workflows
+
+... you want an ORM with a transparent development process, maintenance & support
+
+... you want to be part of an awesome community
+
+
+**Prisma likely is not a good fit for you if ...**
```
Comment (nikolasburk) [inline content/200-concepts/050-overview/250-should-you-use-prisma.mdx]:
```suggestion
**Prisma likely _is not_ a good fit for you if ...**
```

**Pull request merged**